### PR TITLE
Introduce voltage level data model

### LINKED ISSUE #43: AggregatedGridInput does not use VoltageLevel but NodeInput does
I just discovered, that `AggregatedGridInput` as a `setVoltLvl()`-method that allows setting a string. In contrast, `NodeInput` also contains a field `voltLvl` which is of type `VoltageLevel`. @mia-krause can you explain me why we distinguish between this and why this isn't harmonized? I think this is a little bit confusing or even not correctly implemented?!

### LINKED ISSUE #54: Restructure the concept of "voltage level"
Actual information related to a voltage level are now distributed throughout different different places. Therefore, a joint concept is implemented.

### PR DESCRIPTION
**Pull request by ckittl**
Resolves #54 and closes #43. 
I changed from the concept of `enum` for listing common German voltage levels to factory class with static members, as `enum` can only implement `interfaces` and not extend classes.

### CODE REVIEW

**Comment by ckittl** [comment]
!test

**Comment by ckittl** [comment]
!test

File: src/main/java/edu/ie3/models/voltagelevels/GermanVoltageLevelUtils.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          "Niederspannung",
+          Quantities.getQuantity(0.4, KILOVOLT),
+          new HashSet<>(Arrays.asList("lv", "ns")),
+          new RightOpenInterval<>(
+              Quantities.getQuantity(0d, KILOVOLT), Quantities.getQuantity(10d, KILOVOLT)));
+  public static final CommonVoltageLevel MV_10KV =
+      new CommonVoltageLevel(
+          "Mittelspannung (10 kV)",
+          Quantities.getQuantity(10d, KILOVOLT),
+          new HashSet<>(Arrays.asList("ms", "mv", "ms_10kv", "mv_10kV")),
```
Comment (johanneshiry) [inline src/main/java/edu/ie3/models/voltagelevels/GermanVoltageLevelUtils.java]: ❤️

**Comment by johanneshiry** [comment]
!test

**Comment by ckittl** [comment]
!test

**Comment by johanneshiry** [comment]
!test

**Comment by johanneshiry** [comment]
!test

**Comment by johanneshiry** [comment]
!test

**Pull request merged**